CONTRIBUTING.md

```diff
@@ -137,12 +137,11 @@ If you'd like to contribute to the collection of snippets:
 
 * Add a file named main.bicep
   * The test will read this input file for snippet completions. It will take the replacement values listed in this file, replace them with their corresponding placeholder, then ensure Bicep reports no warnings or errors.
+    * E.g. [`res-aks-cluster/main.bicep`](./src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-aks-cluster/main.bicep)
 
-  * E.g. [`res-aks-cluster/main.bicep`](./src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-aks-cluster/main.bicep)
+* Add a file named main.combined.bicep that contains the template combined with placeholder values.
```

Comment:
I think it would be worth pointing out the easiest way to get this file is by running the baseline update script - I imagine most people will not want to handwrite this file.

Reply:
@MarcusFelling , we can add a link to this section https://github.com/Azure/bicep/blob/84b59caf35358e4109c8d8afa362204c236daa80/CONTRIBUTING.md#updating-test-baselines

@anthony-c-martin , would that work?

Reply:
See latest for updates on this